aws_grant_user_access/src/process_event.py

```diff
@@ -14,11 +14,12 @@
 
 
 def process_event(event, policy_creator):
-    validate(instance=event, schema=SCHEMA)
-    time_window = GrantTimeWindow(hours=event["approval_in_hours"])
-    policy_creator.grant_access(
-        role_arn=event["role_arn"],
-        username=event["username"],
-        start_time=time_window.start_time,
-        end_time=time_window.end_time,
-    )
+    for user in event["username"]:
+        validate(instance=event, schema=SCHEMA)
```

Comment:
```suggestion
    validate(instance=event, schema=SCHEMA)
    for user in event["username"]:
```
The call to validate only needs to be made once as it verifies the entire event is correct. It should also be made before accessing data inside the event object. moving it to before the loop should fix this 

Perhaps worth adding a test for this in tests/unit/test_lambda_handler.py

Reply:
thx, addressed in the next commit